README.md

```diff
@@ -1,3 +1,5 @@
+![Tests](actions/workflows/tests.yml/badge.svg)
```

Comment:
```suggestion
![Tests](https://github.com/nheydari/TwitterWatch/actions/workflows/tests.yml/badge.svg)
```
Also, I noticed this doesn't show properly. There's probably a way to make it portable but I'm not sure how. regardless, ^ should fix that.